### DISCUSSION TITLE
Is it possible to reverse the order of the columns in a Grid component?

### DISCUSSION THREAD

**Discussion by piducancore** [system-ui/theme-ui#1864] [Q&A] [answered]
Hey, everyone 👋
I have a `Grid` component with two columns like this:

```jsx
<Grid columns={[ 1, 2 ]}>
    <Box>Column A</Box>
    <Box>Column B</Box>
</Grid>
```

Is it possible to reverse the order of columns at different screen sizes?

**Comment by lachlanjc** [comment, marked as answer]
There's nothing custom in Theme UI for this, but standard CSS will do it:

> ```jsx
> 
> <Grid columns={[ 1, 2 ]}>
> 
>     <Box sx={{ gridColumn: [null, 2] }}>Column A</Box>
> 
>     <Box sx={{ gridColumn: [null, 1] }}>Column B</Box>
> 
> </Grid>
> 
> ```

**Reply by piducancore** [reply]
Thank you 🙏
